### PR TITLE
parser: support show events

### PR DESCRIPTION
**Pull request by XuHuaiyu**
#2015 
This commits support `show events` statement.
PTAL @shenli @coocood @tiancaiamao @hanfei1991 @zimulala @winoros

### CODE REVIEW

**Comment by coocood** [comment]
Add test in `executor/show_test`

**Comment by coocood** [comment]
LGTM

File: parser/parser_test.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (s *testParserSuite) TestSimple(c *C) {
 		"starting", "table", "terminated", "then", "tinyblob", "tinyint", "tinytext", "to",
 		"trailing", "true", "union", "unique", "unlock", "unsigned",
 		"update", "use", "using", "utc_date", "values", "varbinary", "varchar",
-		"when", "where", "write", "xor", "year_month", "zerofill",
+		"when", "where", "write", "xor", "year_month", "zerofill", "events",
```
Comment (tiancaiamao) [inline parser/parser_test.go]: events is unreserved keyword, move it to line `unreservedKws` at line 79

**Comment by shenli** [comment]
@XuHuaiyu Please address the comment.

**Comment by XuHuaiyu** [comment]
@tiancaiamao PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**